avm/res/compute/image/README.md

```diff
@@ -1,10 +1,5 @@
 # Images `[Microsoft.Compute/images]`
 
-> ⚠️THIS MODULE IS CURRENTLY ORPHANED.⚠️
```

Comment:
Waiting for @tony-box membership to relevant avm groups

Reply:
@eriqua - I believe the decision was to not block this in BRM due to group membership and pick up another task in AVM Governance to manage / track ownership?

Reply:
Note that the group membership has been fixed by @segraef, confirmed here: https://github.com/Azure/Azure-Verified-Modules/issues/568

Reply:
@tony-box this is ready to be approved as well :sunglasses: